cmd/mattermost/commands/team.go

```diff
@@ -247,3 +256,34 @@ func listTeamsCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func searchTeamCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
```

Comment:
Instead of this, you can use the "MinimunNArgs" modifier in the init function for this command. See here: https://github.com/spf13/cobra#positional-and-custom-arguments